core/services/validators.go

```diff
@@ -92,9 +93,9 @@ func ValidateInitiator(i models.Initiator, j models.JobSpec) error {
 		return validateExternalInitiator(i)
 	case models.InitiatorServiceAgreementExecutionLog:
 		return validateServiceAgreementInitiator(i, j)
-	case models.InitiatorWeb:
-		fallthrough
 	case models.InitiatorRunLog:
+		return validateRunLogInitiator(i, j)
+	case models.InitiatorWeb:
 		fallthrough
 	case models.InitiatorEthLog:
```

Comment:
I think it'd be more readable to avoid `fallthrough` and simply do
```go
    case models.InitiatorWeb, models.InitiatorEthLog:
        return nil
```